### DISCUSSION TITLE
How to set up smtp for password resets?

### DISCUSSION THREAD

**Discussion by TheBig-O** [Freika/dawarich#265] [Q&A] [answered]
I recently tried sending a reset email to my email address and found that it didn't actually work. When I read through the docs, I didn't see anything about setting up the credentials for sending email. I expected something like environment variables or something in the settings, but have not found anything. Is this a feature that hasn't been implemented yet?

**Comment by Freika** [comment, marked as answer]
Currently, there is no way to set up SMTP config. I might introduce it later, it's not hard.

Meanwhile, you can reset user password (and do so much more) in the Console: https://dawarich.app/docs/FAQ#how-to-update-users-password

**Reply by TheBig-O** [reply]
Thanks for the information. I figured that was the case. I was having issues yesterday and had to rebuild my database. I stumbled across the "forgot password" issue by accident. I'm back in and everything is working just fine but I'll remember the console if I have issues in the future.

**Comment by Freika** [comment]
Done in [0.25.5](https://github.com/Freika/dawarich/releases/tag/0.25.5)
